### ISSUE TITLE
Dataset about backbone

### ISSUE THREAD

**Issue by FengMu1995** [deepinsight/insightface#1928]
I wonder why the backbone was trained with only one dataset every time. Why can't the backbone be trained with multiple dataset.
As you upload "ResNet50@WebFace600K"

**Comment by nttstar** [comment]
What do you mean by multiple datasets?

**Comment by FengMu1995** [comment]
Yes, the multiple datasets are combined into one dataset which be used to train.

**Comment by jacqueline-weng** [comment]
I implemented this function in my code. Make each dataset into the form of mxnet style train.rec and train.idx. When accessing to each index, convert the global index to local index which can be applied to individual dataset. The big assumption is you do not have any overlapping in id between different datasets. It worked pretty well for me.

**Comment by jacqueline-weng** [comment]
> I implemented this function in my code. Make each dataset into the form of mxnet style train.rec and train.idx. When accessing to each index, convert the global index to local index which can be applied to individual dataset. The big assumption is you do not have any overlapping in id between different datasets. It worked pretty well for me.

In this way, you don't have to rewrite training rec each time adding new data.
